src/components/nodes-table/filter-options.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <v-menu
+    :value="show"
+    :close-on-content-click="false"
+    :offset-y="true"
+  >
+    <template v-slot:activator="{ on, attrs }">
+      <v-icon small v-on:click="showOptions()"
+        v-bind="attrs"
+        v-on="on"
+        title="Filter options..."
+      >
+        {{ hasFilter ? 'filter_list_alt' : 'filter_list' }}
+      </v-icon>
+    </template>
+    <v-card>
+      <v-icon small v-on:click="hideOptions()" right>close</v-icon>
+      <v-card-text>
+        <v-row v-if="value && value.type=='string'">
+          <v-col>
+            <v-text-field
+              ref="search"
+              label="Contains"
+              v-model="value.search"
+              clearable
+              @change="hideOptions()"
+              @show="$refs.search.$el.focus()"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+        <v-row v-if="value && value.type=='number'">
+          <v-col>
+            <v-text-field type="number" label="Min" v-model="value.min" clearable></v-text-field>
+          </v-col>
+          <v-col>
+            <v-text-field type="number" label="Max" v-model="value.max" clearable></v-text-field>
+          </v-col>
+        </v-row>
+        <v-row v-if="value && value.type=='date'">
+          <v-col>
+            <v-text-field type="datetime-local" label="From" v-model="value.min" clearable></v-text-field>
+            <v-text-field type="datetime-local" label="Until" v-model="value.max" clearable></v-text-field>
+          </v-col>
+        </v-row>
+        <v-row v-if="value && (value.type=='string' || value.type=='number' || value.type=='boolean')">
```

Comment:
What about use `v-switch` for booleans?

Reply:
I'm not sure, if this will work here, since the secure property filter - while being a boolean - can have multiple states:
* Do not filter
* Filter for true values
* Filter for false values
* Filter for undefined values
That's why I kept v-select here.
Or do you have a better idea.

Reply:
@ahochsteger Aren't states: `indeterminate` `true` and `false`? For filtering bools usually `checkboxes` are used and them can have an `indeterminate` state (that equals to undefined).

Reply:
Check it here: https://vuetifyjs.com/en/components/checkboxes/#states

Reply:
Good idea - will change it to that.

Reply:
A consequence of changing to 3-state v-switch would be that it would not possible anymore to just show nodes where the secure flag is undefined.
Do you still think, it's better to change it from v-select to v-switch?

Reply:
Shouldn't all filter checks skip when switch is `indeterminate` (so the value `undefined` ) ?

Reply:
You are right, that this would be inconsistent with other data types.
The only use case where filtering for undefined values might be for debugging Zwave2Mqtt itself but this may not be something a typical user would need.
If you think we don't need it, I will change it.

Reply:
> If you think we don't need it, I will change it.

Ok!

Reply:
It's done, but I'm not really happy about the resulting layout spacing (caused by v-menu > v-card > v-card-text > v-row > v-col > v-checkbox):
![image](https://user-images.githubusercontent.com/207989/98783968-a7444f00-23fa-11eb-829b-fcf04738649b.png)

I tried to reduce the nesting (at least for v-checkbox) down to v-menu > v-card > v-checkbox but that resulted in missing padding on the left that I was not able to add using CSS spacing helpers (e.g. pa-.. or ma-...) since they had no effect.
Being a first time Vuetify user I could need some help here.
If you have a better suggestion for the checkbox label, please let me know.